gremlin-go/cucumber/featureSteps_test.go

```diff
@@ -286,9 +320,27 @@ func (tg *tinkerPopGraph) theGraphInitializerOf(arg1 *godog.DocString) error {
 		return err
 	}
 	<-future
-	// We may have modified the so-called `empty` graph.
-	if tg.graphName == "empty" {
-		tg.reloadEmptyData()
+	return nil
+}
+
+func (tg *tinkerPopGraph) theResultShouldHaveACountOf(expectedCount int) error {
+	actualCount := len(tg.result)
+	if actualCount == 1 {
+		switch reflect.TypeOf(tg.result).Kind() {
+		case reflect.Slice, reflect.Array:
+			if actualCount == 1 {
+				result := tg.result[0].(*gremlingo.Result).GetInterface()
+				switch reflect.TypeOf(result).Kind() {
+				case reflect.Slice, reflect.Array:
+					actualCount = len(result.([]interface{}))
```

Comment:
```suggestion
					actualCount = result.Len()
```

Reply:
This code block is actually unused and I've removed it.